clients/hipfft_params.h

```diff
@@ -1184,14 +1211,23 @@ class hipfft_params : public fft_params
             break;
         }
 
+        auto* inembed = ll_inembed.data();
+        auto* onembed = ll_onembed.data();
+        if(is_using_default_layout())
+        {
+            // test hipfft's ability to figure it out
+            inembed = nullptr;
+            onembed = nullptr;
+        }
```

Comment:
It seems that this only checks that hipfft can populate i/onembed for the default layout case.  If we really want to check it, we should provide more complicated data layouts (eg stride != 1 should give reasonable nembed values).  Would adding specific tests be more effective?  For example, if we make two plans, one specifying a trivial nembed, and the other not, then we should be able to check that the plans are identical.